checklist/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "amp-checklist-bot",
```

Comment:
High-level nit: let's see if we can figure out a clearer name

Reply:
`amp-contribution-checklist-bot` seems unambiguous enough but rather verbose to me. Up to you.

Reply:
Eh I guess you're right, checklist bot is good enough